README.md

```diff
@@ -13,29 +13,32 @@ This exports **Clients** and Typescript **typings** to help you connecting a VTE
 | Catalog         | `getSkuById`                                                |
 | Logistics       | `getDockById`, `pickupById`, `nearPickupPoints`, `shipping` |
 | OMS             | `userLastOrder`, `order`                                    |
+| Checkout        | `getOrderFormConfiguration`, `setOrderFormConfiguration`, `setSingleCustomData` |
+| OMS Proxy       | `orders`, `orderFormId`, `customData`, `register`           |
+---
 
-
+In order to know a little bit more about the methods that the `OMS Proxy` client implements, you can check out [this document](https://www.notion.so/How-to-use-the-OMS-API-Proxy-application-e82f11ff896247c58a7e2e658d631516). It's important to emphasize that to use this client, you will have to declare a dependency and a policy, this is also describe in the document mentioned above.
```

Comment:
I'd suggest adding a column on the table above and linking this resource there. If we add some other clients, both data will be apart on the document.

I think we can change this later with a proposal for documentation (I'm thinking a table for each client with a reserved section)